src/test/java/com/google/cloud/genomics/dataflow/utils/VariantUtilsTest.java

```diff
@@ -34,7 +34,16 @@
   public void testIsVariant() {
     assertTrue(VariantUtils.isVariant(DataUtils.makeVariant("chr7", 200000, 200001, "A",
```

Comment:
I see what you're trying to do.  So we can assume things such as the following will be processed correctly, right?

```
1  2337032 rs1129171   C   T,<NON_REF> 480.77  .   BaseQRankSum=0.218;ClippingRankSum=0.103;DB;DP=45;MLEAC=1,0;MLEAF=0.500,0.00;MQ=60.00;MQ0=0;MQRankSum=-1.344;ReadPosRankSum=1.046   GT:AD:DP:GQ:PL:SB   0/1:19,26,0:45:99:509,0,330,565,407,97
2:9,10,13,13
```